doxygen-public.conf

```diff
@@ -905,7 +905,7 @@ RECURSIVE              = NO
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                =
+EXCLUDE                = ./src/audio ./src/fatfs
```

Comment:
When I revamped doxygen, I didn't realize that the option `RECURSIVE=OFF` meant that `src/audio` wasn't being processed by doxygen. The mixer docs were still be present because the header files are in the `include` dir, but the C files weren't being processed. In this PR, I'm activating `RECURSIVE=ON` to document `src/rspq/*.c`, but that causes a CI failure because `src/audio` is missing some docs now. I need to add missing docs there as well.

Reply:
Can this be resolved?